### PR TITLE
(DOCSP-26742): Flutter custom user data

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

document using access token in Flutter SDK

### Jira

- https://jira.mongodb.org/browse/DOCSP-26742

### Staged Changes

- [Get the User Access Token](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-26742/sdk/flutter/users/access-token)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/flutter/users/access-token.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Get the User Access Token - Flutter SDK
+=======================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Every User object contains a JWT token which you can use to access Atlas App Services.
```
Comment (cbush) [inline source/sdk/flutter/users/access-token.txt:13]:
```suggestion
Every User object contains a JWT token that you can use to access Atlas App Services.
```

File: source/sdk/flutter/users/access-token.txt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+You can use the access token to query the Atlas GraphQL API from your client application
```
Comment (cbush) [inline source/sdk/flutter/users/access-token.txt:15]: This sentence could be reworked or broken up. There are two ideas here: "you can use the token to do what" and "using a GraphQL client, like...". You can use the token with any GraphQL client to query the Atlas GraphQL API.

File: source/sdk/flutter/users/access-token.txt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+<realm/User/accessToken.html>` property.
+
+.. literalinclude:: /examples/generated/flutter/access_token_test.snippet.get-access-token.dart
+   :language: dart
+
+Refresh the Access Token
+------------------------
+
+The access token expires 30 minutes after a user logs in. It does not refresh automatically.
+You can refresh it with :flutter-sdk:`User.refreshCustomData() <realm/User/refreshCustomData.html>`.
```
Comment (cbush) [inline source/sdk/flutter/users/access-token.txt:33]:
```suggestion
Refresh it with :flutter-sdk:`User.refreshCustomData() <realm/User/refreshCustomData.html>`.
```

File: source/sdk/flutter/users/access-token.txt [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+.. literalinclude:: /examples/generated/flutter/access_token_test.snippet.refresh-access-token.dart
+   :language: dart
+
+You can also periodically refresh the access token to make sure it's always up to date
```
Comment (cbush) [inline source/sdk/flutter/users/access-token.txt:38]:
```suggestion
You can also periodically refresh the access token 
```

**Review by cbush — APPROVED** [review]
Looks good. A few wording nitpicks.

File: source/sdk/flutter/users/access-token.txt [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Every User object contains a JWT token which you can use to access Atlas App Services.
+
+You can use the access token to query the Atlas GraphQL API from your client application
+using any GraphQL client, like `graphql_flutter <https://pub.dev/packages/graphql_flutter>`__.
```
Comment (mongodben) [inline source/sdk/flutter/users/access-token.txt:16]:
```suggestion
You can use the access token to query the Atlas GraphQL API from your client application.
Use any GraphQL client to query the Atlas GraphQL API, such as `graphql_flutter <https://pub.dev/packages/graphql_flutter>`__.
```

File: source/sdk/flutter/users/access-token.txt [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+with the timer's callback function.
+
+.. literalinclude:: /examples/generated/flutter/access_token_test.snippet.periodic-refresh-access-token.dart
+   :language: dart
+
+
+
+
+
+
```
Comment (mongodben) [inline source/sdk/flutter/users/access-token.txt:50]:
```suggestion
```

**Pull request merged**